### ISSUE TITLE
Clear email hash using null not working

### ISSUE THREAD

**Issue by RareData** [ripple/ripple-lib#965]
Setting an email hash using the prepareSettings and sign methods work fine, but following the API documentation to clear the email hash doesn't.

Reproduce the issue by preparing a settings transaction using `api.prepareSettings({emailHash: null}, {sequence: 3, fee: "0.00012", maxLedgerVersion: null})`. The sequence number must be adjusted.

The response should look something like:
```{txJSON: "{"TransactionType":"AccountSet","Account":"rHehtyAwASiwsrD6W5xWDkw2tSS6cicuGy","EmailHash":"0","Flags":2147483648,"Fee":"12","Sequence":3}", instructions: {fee:"0.000012", maxLedgerVersion: null, sequence:3}}```

Trying to perform `api.sign(txJSON, secret)` with this prepared settings transaction, results in the following assertion error (stack):

```"AssertionError: 1 == 16
    at klass.Hash [as inherits] (webpack-internal:///./node_modules/ripple-binary-codec/distrib/npm/types/hash.js:11:12)
    at new klass (webpack-internal:///./node_modules/ripple-binary-codec/distrib/npm/utils/make-class.js:29:16)
    at Function.from (webpack-internal:///./node_modules/ripple-binary-codec/distrib/npm/types/hash.js:20:14)
    at eval (webpack-internal:///./node_modules/ripple-binary-codec/distrib/npm/types/st-object.js:31:46)
    at eval (webpack-internal:///./node_modules/lodash/lodash.js:13805:16)
    at eval (webpack-internal:///./node_modules/lodash/lodash.js:4925:15)
    at baseForOwn (webpack-internal:///./node_modules/lodash/lodash.js:3010:24)
    at Function.transform (webpack-internal:///./node_modules/lodash/lodash.js:13804:43)
    at Function.from (webpack-internal:///./node_modules/ripple-binary-codec/distrib/npm/types/st-object.js:28:18)
    at serializeObject (webpack-internal:///./node_modules/ripple-binary-codec/distrib/npm/binary.js:23:18)"```

**Comment by RareData** [comment]
You can currently only clear by setting `{emailHash: "00000000000000000000000000000000"}`, not sure if this is the intended behaviour.

**Comment by RareData** [comment]
Fixed in ripple-lib 1.4.2
